SU-SWS/stanford_profile
# D8CORE-1221 Dark paragraph type icons

# READY FOR REVIEW

# Summary
- Darker Icons

# Need Review By (Date)
- 2/12

# Urgency
- low

# Steps to Test
1. checkout branch
1. `drush cim`
1. review icons on the paragraph types to ensure they are darker

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
